packages/rich-text-editor/test/basic-lit.test.js

```diff
@@ -1,3 +1,2 @@
-import '../theme/lumo/vaadin-rich-text-editor-styles.js';
-import '../src/vaadin-lit-rich-text-editor.js';
+import '../theme/lumo/vaadin-lit-rich-text-editor.js';
```

Comment:
Is this related?

Reply:
There is an error (which only happens in Polymer version though) with the change:

```
packages/rich-text-editor/test/basic-polymer.test.js:

 🚧 Browser logs:
      Error: The trap node should have at least one focusable descendant or be focusable itself
```

This looks like a timings issue. I think this change can be extracted to separate PR though.

Reply:
Actually, current setup is incorrect since it uses Lumo version of RTE + unstyled version of dialog / text-field.

Reply:
Reported #7249 to fix the test. I will merge the current PR as is to make it easier to backport the fix.